src/Apps/MySQLWorkbenchApp.php

```diff
@@ -65,7 +64,7 @@ public function validate() {
         return FALSE;
     }
 
-    return $this->which($this->app_location);
+    return file_exists($this->app_location);
```

Comment:
Can you undo this and the change to app_location and see if it fixes it? $this->which should be fixed from here: https://github.com/terminus-plugin-project/terminus-pancakes/commit/7a3dd950216edab4438a2e2812706e709db4f6b1